### PR TITLE
Update gcp-search extension

### PR DESCRIPTION
**Pull request by hisomura**
## Description

- Update product list
- Update npm packages
- Update tsconfig.json and .eslintrc.json

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/google-cloud-platform-search/src/consoleProducts.tsx
Comment (hisomura) [inline extensions/google-cloud-platform-search/src/consoleProducts.tsx:1]: I executed a following script on [All Products page](https://console.cloud.google.com/products) to obtain product list.
```js
const elements = [...document.getElementsByClassName('product-name')]
elements.map(element => console.log(`{ name: "${element.innerText}",  toUrl: (projectId) => \`${element.parentElement.href}\` },`))
```

File: extensions/google-cloud-platform-search/package.json [hunk 1]
```diff
@@ -26,24 +27,25 @@
...
-    "react-devtools": "^4.19.2",
-    "typescript": "^4.4.3"
+    "@raycast/eslint-config": "^1.0.6",
+    "@types/node": "20.8.10",
+    "@types/node-localstorage": "^1.3.3",
+    "@types/react": "18.2.27",
+    "eslint": "^8.51.0",
+    "prettier": "^3.0.3",
+    "react-devtools": "^4.10.4",
+    "typescript": "^5.2.2"
```
Comment (hisomura) [inline extensions/google-cloud-platform-search/package.json:42]: I updated the devDependencies based on the new project creation template.
https://developers.raycast.com/basics/create-your-first-extension

**Comment by pernielsentikaer** [comment]
Hi @hisomura 👋

Thanks for your contribution 🔥

@klaa97 - could you check this?

**Review by klaa97 — APPROVED** [review]
Seems all good, thank you!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**